include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -474,14 +474,14 @@ struct GemmKernel
      * @tparam DstInMemOp Destination memory operation (default: set).
      */
     template <memory_operation_enum DstInMemOp = memory_operation_enum::set>
-    CK_TILE_DEVICE static void RunGemm(const ADataType* a_ptr,
-                                       const BDataType* b_ptr,
-                                       CDataType* c_ptr,
-                                       void* smem_ptr,
-                                       const GemmKernelArgs& kargs,
-                                       const SplitKBatchOffset& splitk_batch_offset,
-                                       const index_t block_idx_m,
-                                       const index_t block_idx_n)
+    CK_TILE_DEVICE static void RunGemmSinglePointer(const ADataType* a_ptr,
```

Comment:
This name is really confusing. My first thought was that there'd be multiple tensors pointers for each input... 
```suggestion
    CK_TILE_DEVICE static void RunGemm(const ADataType* a_ptr,
```